{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py

```diff
@@ -42,8 +42,35 @@ def get_tm_session(session_factory, transaction_manager):
           with transaction.manager:
               dbsession = get_tm_session(session_factory, transaction.manager)
 
+    This function may be invoked with a ``request`` kwarg, such as when invoked
+    by the reified ``request.dbsession`` Pyramid Request attribute. The default
+    value, for backwards compatibility, is ``None``.
+
+    - The ``request`` kwarg is used to populate the SQLAlchmey Session's "info"
+      dict.  The "info" dict is the official namespace for developers to stash
+      Session-specific information.  For more information, please see the
+      SQLAlchemy docs:
+      https://docs.sqlalchemy.org/en/13/orm/session_api.html?#sqlalchemy.orm.session.Session.params.info
+
+    - By placing the active Request in the "info" dict, developers will be able
+      to access the active Pyramid Request from an instance of a SQLAlchemy
+      object in one of two ways:
+
+    - Classic SQLAlchemy. This uses the Session's utility classmethod:
+
+        from sqlalchemy.orm.session import Session as sa_Session
+
+        dbsession = sa_Session.object_session(dbObject)
+        request = dbsession.info["request"]
+
+    - Modern SQLAlchemy. This uses the Runtime API:
```

Comment:
Why is Runtime capitalized?

Reply:
I will drop this to "runtime".  The proper name in SQLAlchemy docs is "Runtime Inspection API", which presents "runtime" information.